src/index.ts

```diff
@@ -606,7 +604,21 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     }
   }
 
-  return { on, off, variation, close, setEvaluations, registerAPIRequestMiddleware, refreshEvaluations }
+  return {
+    on,
+    off,
+    close,
+    setEvaluations,
+    registerAPIRequestMiddleware,
+    refreshEvaluations,
+    variation: (identifier: string, defaultValue: any) => {
```

Comment:
Not sure how you're gonna feel about using closures here @knagurski  - it was the only way I could find to keep the same `Result` object unchanged.